### PR TITLE
Update PhotoSubmissionAction.js

### PR DESCRIPTION
**Pull request by Dinnall**
### What's this PR do?

This pull request updates copy on the photo uploader when someone is adding a new photo to make the quantity ask clearer

### How should this be reviewed?

...

### Any background context you want to provide?

...

### Relevant tickets

References [Pivotal # 174319898](https://www.pivotaltracker.com/n/projects/2401401/stories/174319898).

Change to :
Screen Shot 2020-09-25 at 6 24 46 PM

Changed from:
Screen Shot 2020-08-13 at 10 59 35 AM (1)

### Checklist

- [ x ] This PR has been added to the relevant Pivotal card.
- [ x ] Added screenshots of front-end changes.

### CODE REVIEW

**Comment by katiecrane** [comment]
@Dinnall could you add screenshots of what it looks like now? thanks!

**Comment by Dinnall** [comment]
> @Dinnall could you add screenshots of what it looks like now? thanks!

Sure! I updated the PR description with a screenshot

> @Dinnall could you add screenshots of what it looks like now? thanks!

File: resources/assets/components/actions/PhotoSubmissionAction/PhotoSubmissionAction.js [hunk 1]
```diff
@@ -423,7 +423,7 @@ class PhotoSubmissionAction extends PostForm {
                             htmlFor="quantity"
                           >
                             {quantity
-                              ? 'You can enter your new total here:'
+                              ? 'Total Items - Updated (this is the total items number above plus the new number you are adding. For instance, if you currently have 1 total item and are adding 1 more, you would enter 2 here):'
```
Comment (katiecrane) [inline resources/assets/components/actions/PhotoSubmissionAction/PhotoSubmissionAction.js:426]: The `T` in `This` should be capitalized like this:
```suggestion
                              ? 'Total Items - Updated (This is the total items number above plus the new number you are adding. For instance, if you currently have 1 total item and are adding 1 more, you would enter 2 here):'
```

**Comment by DFurnes** [comment]
Holy form label! Have we considered using help text for the additional explanation, like this?

Screen Shot 2020-09-25 at 11 26 10 AM

**Comment by Dinnall** [comment]
> form label
I don't think that format was considered from my knowledge. 
Do you suggest switching how the order of description and the quantity input are displayed?

**Comment by DFurnes** [comment]
I'd be curious to run it by folks on the product team as an option, but certainly doesn't need to block merging.

cc: @LeahAngle @ddonizeth

**Comment by ddonizeth** [comment]
@DFurnes I LOVE how this looks like now!
The information looks much better organized, with a clear distinction between what is the title and what is the explanation
Let me bring Leah's attention to this PR so she can also share her thoughts!

**Comment by LeahAngle** [comment]
I love using the helper text! I didn't know we had this option! I'm going to update our design files to include. Thank you @DFurnes for this great suggestion!

**Comment by DFurnes** [comment]
Awesome, glad I asked! 😸 

@Dinnall I mocked that up by copying the styles we'd used on the little character counter below and placing it right below the form field inside the `div`. It uses a `p` tag with the `text-sm text-gray-600` Tailwind classes.

**Comment by Dinnall** [comment]
> Awesome, glad I asked! 😸
> 
> @Dinnall I mocked that up by copying the styles we'd used on the little character counter below and placing it right below the form field inside the `div`. It uses a `p` tag with the `text-sm text-gray-600` Tailwind classes.

Got it! Very good suggestion. I went ahead and updated the code and PR screenshot

cc: @ddonizeth @LeahAngle

**Review by DFurnes — APPROVED** [review]
Looks perfect!

**Pull request merged**